app/features/feedback/lab/containers/feedback-section-container.jsx

```diff
@@ -0,0 +1,78 @@
+import React, { Component, PropTypes } from 'react';
+import ModalFormDialog from 'modal-form/dialog';
+import _ from 'lodash';
+import FeedbackSection from '../components/feedback-section';
+import RuleEditorModal from './rule-editor-modal-container';
+
+class FeedbackSectionContainer extends Component {
+  constructor(props) {
+    super(props);
+    this.deleteRule = this.deleteRule.bind(this);
+    this.openRuleEditorModal = this.openRuleEditorModal.bind(this);
+    this.saveRule = this.saveRule.bind(this);
+    this.saveTaskChanges = this.saveTaskChanges.bind(this);
+  }
+
+  deleteRule(ruleId) {
+    const editedTask = _.assign({}, this.props.task);
```

Comment:
Is there a reason to use lodash's assign over the native Object.assign()?

Reply:
Not really - I was in the habit of using lodash for all the things at that point, and it's a little terser. I don't mind swapping it out, if you'd prefer?

Reply:
Ooh, side note (although it doesn't really apply because I think it's polyfilled anyway) - `_.assign()` is basically a polyfill for IE, it defers to `Object.assign()` if it's available.